### PR TITLE
Use appmanager instead of OC_App for check for enabled app

### PR DESCRIPTION
**Pull request by MorrisJobke**
Preparation for a cleanup of legacy classes

### CODE REVIEW

File: core/Command/Maintenance/Repair.php [hunk 1]
```diff
@@ -78,9 +78,10 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			}
 		}
 
-		$apps = \OC::$server->getAppManager()->getInstalledApps();
+		$appManager = \OC::$server->getAppManager();
```
Comment (icewind1991) [inline core/Command/Maintenance/Repair.php]: Bonus points when doing proper DI here

**Review by icewind1991 — APPROVED** [review]
Good otherwise

File: lib/public/App.php [hunk 2]
```diff
@@ -127,9 +127,10 @@ public static function getAppInfo( $app, $path=false ) {
 	 *
 	 * This function checks whether or not an app is enabled.
 	 * @since 4.0.0
+	 * @deprecated 13.0.0 use \OC::$server->getAppManager()->isEnabledForUser($appId)
 	 */
 	public static function isEnabled( $app ) {
-		return \OC_App::isEnabled( $app );
```
Comment (ChristophWurst) [inline lib/public/App.php]: Any objections on deprecating https://github.com/nextcloud/server/blob/18601903052e389d7dbc25ff2603eb7091415ca0/lib/private/legacy/app.php#L346-L356 too?

**Review by ChristophWurst — COMMENTED** [review]
Looks good. Found another static legacy method we could deprecate.

**Review by blizzz — APPROVED** [review]

File: lib/public/App.php (hunk 2 above)
Comment (MorrisJobke) [reply lib/public/App.php]: I will take care of it in a followup PR.

Comment (ChristophWurst) [reply lib/public/App.php]: Thanks

File: core/Command/Maintenance/Repair.php (hunk 1 above)
Comment (MorrisJobke) [reply core/Command/Maintenance/Repair.php]: I will do this in a followup PR.

**Comment by MorrisJobke** [comment]
All requested changes are now implemented in a dedicated PR.

**Pull request merged**